e2eAIOK/ModelAdapter/src/task/medical_segmentation/README.md

```diff
@@ -0,0 +1,30 @@
+## Environment Setup
+
+1. build docker image
+   ```
+   cd Dockerfile-ubuntu18.04 && docker build -t aidk-pytorch110 . -f DockerfilePytorch110 && cd .. && yes | docker container prune && yes | docker image prune
```

Comment:
please remove `aidk` typos and use e2eaiok

Reply:
Updated